sass/_theme_badge.sass

```diff
@@ -35,7 +35,8 @@
       color: $black
   &.shift-up
     height: auto
-    max-height: 100%
+    max-height: 90%
```

Comment:
What is the reason for this line? Why not let it take 100% height?

Reply:
With `max-height: 100%`, this happens:

![ezgif-3-b0f25b01dc0b](https://user-images.githubusercontent.com/29149191/50148672-1bdde400-02df-11e9-9e7e-68e2016ee61d.gif)

Reply:
OK, I see. The height of 90% is fine for this example where the rtd theme is not used, but just the badge_only CSS. Here, the versions popup is not aligned to the bottom of the window.
When the rtd theme is used, the versions popup is aligned to the bottom of the window and a  height of 100% would be better.

Can you try keeping `max-height: 100%` on `&.shift-up` as it is, but setting `max-height: 90%` on `&.rst-badge` in `_theme_badge.sass:60`?

Reply:
That works.
I pushed the changes.